libcodechecker/libauth/cc_ldap.py

```diff
@@ -312,11 +312,9 @@ def auth_user(ldap_config, username=None, credentials=None):
                               account_scope)
 
         if user_dn is None:
-            LOG.error('Please check your LDAP server '
-                      'authentication credentials.')
+            LOG.error("Invalid credentials for user: '{0}'!".format(username))
```

Comment:
Please use a different logging format for optimization. You read about it [here](https://docs.python.org/2.6/library/logging.html#optimization)